components/image/Image.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { SHOW_IMAGES } from 'proton-shared/lib/constants';
+import Button from '../button/Button';
+import useMailSettings from '../../hooks/useMailSettings';
+
+const Image = (props) => {
+    const [{ ShowImages }] = useMailSettings();
```

Comment:
⚠️ `MailSettings` are not loaded everywhere. You need to handle `loading`, the second parameter.